app/assets/javascripts/html5shiv.js

```diff
@@ -0,0 +1,335 @@
+/**
```

Comment:
Was committing the `html5shiv` intentional?

Reply:
See my comment on #18 - `npm install` brings the file in.